content/posts/addons-by-default.md

```diff
@@ -0,0 +1,100 @@
+Since then, all projects building with MkDocs are already using Read the Docs Addons and they haven't reported any big issues.
+
+All of this step-by-step changes have given us the confidence to make the last step and **plan the removal of the config manipulation for Sphinx projects** as well,
+which is the most used documentation tool on Read the Docs,
+and that's why we left it for the end.
+
+
+## Deprecation timeline
+
+We understand this change is going to affect _all the projects_ building on Read the Docs,
+so we have a timeline to make this deprecation easier for our users.
+Keep these dates in mind and prepare your projects for the changes:
+
+- **July 29, 2024**: enable Read the Docs Addons for _all **new** projects_ created.
+- **October 7, 2024**: enable Read the Docs Addons for _**all projects**_.
+
+
+## How does it affect my projects?
+
+It's worth to note there are two big different types of changes that will affect your projects:
+at build time, and visually.
+Depending on the configuration of your project, it could be affected by one, both or none of them.
+
+### At build time changes
+
+As we mentioned before, all the Sphinx's `conf.py` file manipulation won't happen anymore at build time.
+This means that if your project were using any of these extra configuration,
+those won't be available at build time anymore and some features may not work as expected,
+or the build could fail completely.
+
+In case you've found any difference in behavior after Read the Docs Addons has been enabled
+(see [How to opt-in to addons _now_](#how-to-opt-in-to-addons-now) to enable addons manually),
+you can try installing the [`sphinx-build-compatibility`](https://github.com/readthedocs/sphinx-build-compatibility/) extension we've created
+to allow keeping the "behind the scenes" manipulation behavior at build time while you can dig more on these changes and perform the migration gradually.
+
+### Visual changes
+
+Even if the build has completed successfully, there may be some visual changes you noticed.
+Version and pull request notifications were completely re-designed and the flyout menu has been moved to a floating position in the bottom right,
+for example.
+
+Depending on your project's configuration, the documentation tool and the theme you are using, you will notice these differences.
+We plan to release a new version of our theme to keep the flyout menu at the bottom left integrated with the navbar soon,
+and we are talking to other theme authors to communicate these changes and work with them towards a better integration.
+Expect news around this visual changes in the following months.
+
+
+## How to opt-in to addons _now_
+
+Read the Docs Addons is already available for all users that want to start using it on their projects.
+In case they want to prepare themselves for the migration, or make usage of the new features from Read the Docs Addons, like the [DocDiff](https://docs.readthedocs.io/page/pull-requests.html) for example.
```

Comment:
```suggestion
In case they want to prepare themselves for the migration, or make usage of the new features from Read the Docs Addons, like the [DocDiff addon](https://docs.readthedocs.io/page/pull-requests.html).
```